### DISCUSSION TITLE
Order field

### DISCUSSION THREAD

**Discussion by JuanPabloDiaz** [rupali-codes/LinksHub#2292] [Ideas]
I was wondering if we could add an **`Order`** field in the project to rate the quality of a resource.

Finding the right resource among a bunch of options can be tricky, especially when it comes to quality assessment.

order*: `number - required`
- Number between 0 and 100 that represents the quality of the resource. Being 0 bad and 100 excellent.
- To rate the quality of a resource, you can use criteria like: UI/UX design, how permissive the license is (CC0 is great, needing to attribute or not use in commercial projects not so much), is it completely free or just some elements, etc.

**Comment by Anmol-Baranwal** [comment]
@JuanPabloDiaz, 
This is currently being worked on. 
After the next release, you will have an upvote button available, allowing anyone to upvote it — similar to a like button -> to recommend a resource. Plus, users will be able to save their resources.

We are implementing GitHub OAuth for this feature. 
Of course, users can still use LinksHub without logging in, but in that case, these options will not be available to them.

Furthermore, manually rating resources from 0 to 100 is not accurate, as the quality of a resource can vary based on personal opinion. We are providing different values of worth which is not very optimal way in my opinion. Plus, it will be very hard due to manual process.

**Comment by JuanPabloDiaz** [comment]
That is awesome!
Thanks for sharing. 

Please, let me know if I can help.

**Reply by JuanPabloDiaz** [reply]
One of the criteria I was considering and was planning on using to sort the resources could be the number of GitHub Stars they had. This could help identify the most popular and reliable ones among the many options available.

**Comment by aftabrehan** [comment]
> One of the criteria I was considering and was planning on using to sort the resources could be the number of GitHub Stars they had. This could help identify the most popular and reliable ones among the many options available.

@JuanPabloDiaz, Not all resources have a corresponding GitHub repository, so relying solely on GitHub Stars as a sorting criterion wouldn't be feasible.

**Reply by JuanPabloDiaz** [reply]
True
